apps/src/templates/curriculumCatalog/cardImageNamesToSrc.js

```diff
@@ -0,0 +1,3 @@
+export const cardImageNamesToSrc = {
+  'another-hoc': require('@cdo/static/resource_cards/anotherhoc.png')
```

Comment:
I just added an already-existing image here. We'll need a more robust strategy here once we have a sense of which courses to use. This pattern is from https://github.com/code-dot-org/code-dot-org/blob/21b824ad1d5982f1cf8d6b3c1109896891f299e6/apps/src/templates/VerticalImageResourceCard.jsx#L54-L82

Reply:
The card images are set as a field on Course Offering now. Most of the Course Offerings should have images. Will that help with this?

Reply:
Ah, good call! That's great –– those fields will be the keys here. The values (e.g. `require(@cdo/static/.../../.png)` will need to be hard-coded because we can't dynamically require assets (as far as I know). But we could still probably put this in a shared constants file for validation on both the front-end and the back-end –– will work with Turner on this and add it to standing to-dos. Thanks!

Reply:
I would expect these images to come from the server as urls pointing to images.code.org, where the image will already be hosted, so you shouldn't need to pull any images into this file. See this PR for details:
* https://github.com/code-dot-org/code-dot-org/pull/50777

In case you do encounter `require()` again, I would add that we are trying to move away from that syntax, so it's better to use `import` instead where possible. if you do need to dynamically load something, you can do it with `import()`. here is one example: https://github.com/code-dot-org/code-dot-org/blob/24c777c9e93a8b7604b8734119dd91c716e7761e/apps/src/imageUtils.js#L73

Reply:
...but if this is just a placeholder until you get the values from the server then it seems ok to use temporarily.  in that case, I would love if you could include a note in the Jira ticket indicating that you'll remove all the `require()` syntax before launch, presumably when you start pulling these values from the server (subtask 4).

Reply:
Good call. This is a placeholder, but was setting up for more of this –– I've modified the pattern with comments to make it clear that this is a temporary solution for storybook